### ISSUE TITLE
KanbanBoard project file has both : EntityFrameworkCore and SQLitePCLRaw entries

### ISSUE THREAD

**Issue by davefxy** [VladislavAntonyuk/MauiSamples#90]
KanbanBoard project file has both : EntityFrameworkCore and SQLitePCLRaw entries. I dont know if this was on purpose with the intention of requiring the user to delete the SQLitePCLRaw entries because the project  wont compile otherwise. The project files themselves reference Microsoft.EntityFrameworkCore.

**Comment by VladislavAntonyuk** [comment]
EFCore has dependencies on that libs, the project compiles without errors. Could you provide more details?

**Comment by davefxy** [comment]
You are right. I havent tried building an iOS app yet. I dont have a Mac so I build only PC and iOS. Thank you for pointing that out dependencies for iOS/MacCatalyst builds.
I have enjoyed perusing your code. I have learned about setting up the databases. I wasnt aware of builder.Services.AddDbContext.
